contracts/crosschain/README.adoc

```diff
@@ -0,0 +1,38 @@
+= Crosschain
+
+[.readme-notice]
+NOTE: This document is better viewed at https://docs.openzeppelin.com/community-contracts/proxy
+
+Gateways are contracts that enable cross-chain communication. These can either be a message source or a destination according to ERC-7786, which defines the following interfaces:
+
+ * {IERC7786GatewaySource}: A contract interface to send a message to another contract on a different chain.
+ * {IERC7786Receiver}: An interface that allows an smart contract to receive a crosschain message provided by a trusted destination gateway.
+
+The library provides an implementation of an ERC-7786 receiver contract as a building block:
+
+ * {ERC7786Receiver}: ERC-7786 cross-chain message receiver.
+
+Given ERC-7786 could be enabled natively or via adapters. Developers can access interoperability protocols through gateway adapters. The library includes the following gateway adapters:
+
+ * {AxelarGatewayBase}: Gateway adapter implementation for the https://www.axelar.network/[Axelar Network].
```

Comment:
```suggestion
 * {AxelarGatewayBase}: Core gateway logic for the https://www.axelar.network/[Axelar] adaptor.
```

Reply:
I'm adding this commit with "adapter" instead. I remember we agreed using us english